tests/bench/coremark/coremark_common.h

```diff
@@ -23,10 +23,16 @@ Original Author: Shay Gal-on
 /* Configuration: TOTAL_DATA_SIZE
         Define total size for data algorithms will operate on
 */
-#ifndef CONFIG_COREMARE_DATA_SIZE
 #ifndef TOTAL_DATA_SIZE
 #define TOTAL_DATA_SIZE 2 * 1000
 #endif
+
+#ifdef CONFIG_COREMARK_FORCE_VERY_QUICK
+#ifdef TOTAL_DATA_SIZE
+#undef TOTAL_DATA_SIZE
+#endif
+#define TOTAL_DATA_SIZE 400
+#define CONFIG_COREMARK_ITERATION_CNT 1
```

Comment:
Please avoid defining Kconfig macros in a header other than autoconf.h to mess up the orders.

Reply:
> Please avoid defining Kconfig macros in a header other than autoconf.h to mess up the orders.

fixed